### DISCUSSION TITLE
delete features from a layer and detect click?

### DISCUSSION THREAD

**Discussion by TobiH-GE** [Mapsui/Mapsui#1913] [Q&A]
how to delete features from a layer and how to detect a click on them? i'm using 4.0.0-beta8 and MAUI.

**Comment by pauldendulk** [comment]
The layer needs to have `layer.IsMapInfoLayer = true;` and you need to listen to the Map.Info event. The `MapInfoEventArgs.MapInfo.Feature` will contain the clicked feature. There is also the Mapsui.Samples.Wpf.Editing sample that has a way to delete items.

**Comment by charlenni** [comment]
There are to few information in your question. So precise it: add which type of control if Mapsui do you use and how you create the features in which type of layer. Perhaps you could look into the samples.
